prow/repoowners/repoowners.go

```diff
@@ -736,8 +737,35 @@ func (o *RepoOwners) applyOptionsToPath(path string, opts dirOptions) {
 	}
 }
 
-func (o *RepoOwners) filterCollaborators(toKeep []github.User) *RepoOwners {
-	collabs := sets.New[string]()
+// Extend github team to appproves/reviewers list
+// If appproves/reviewers name contains '/', like 'myorg/developer'
+// It will be recognized as github team.
+func extendGithubTeam(ars sets.String, ghc githubClient) sets.String {
+	newArs := sets.NewString()
+	for ar := range ars {
+		if strings.Contains(ar, "/") {
+			arInfo := strings.Split(ar, "/")
+			if len(arInfo) == 2 {
+				org := arInfo[0]
+				teamName := arInfo[1]
+				members, err := ghc.ListTeamMembersBySlug(org, teamName, "all")
+				if err != nil {
+					logrus.Warnf("Get github team[%s] members failed: %v", ar, err)
```

Comment:
pls do `WithError` and `WithValue` to make the log structured instead of a raw string